ports/servoshell/webview.rs

```diff
@@ -412,11 +413,11 @@ where
         let mut need_present = false;
         let mut history_changed = false;
         for (webview_id, msg) in events {
-            trace!(
-                "embedder <- servo EmbedderMsg ({:?}, {:?})",
-                webview_id.map(|x| format!("{}", x)),
-                msg
-            );
+            if let Some(webview_id) = webview_id {
+                trace_from_servo!(msg, "{webview_id} {msg:?}");
```

Comment:
Simillarly "trace_embedder_msg" ? Dunno. Maybe these are more complicated than they seem?

Reply:
done in 198decce710d39b80ab8e02b35396515145186b5